### DISCUSSION TITLE
Multiple drivers

### DISCUSSION THREAD

**Discussion by OJPARKINSON** [br-g/openf1#94] [Q&A] [answered]
Would it be possible to query by multiple drivers? So in theory I could get the stint information from a session that has driver 16,44,4 and so on.

**Comment by adityakotha03** [comment, marked as answer]
The best bet is to pull all data at once and then use whatever driver stint you want. Not the optimal strategy, but it gets the work done.

**Reply by OJPARKINSON** [reply]
I'd be happy to look at making a PR for the work, just interested if there is an issue with having that functionality?

**Reply by adityakotha03** [reply]
I can't comment anything on it as I haven't looked at the code myself, maybe @br-g could help you out with this.

**Comment by y512h** [comment]
how to get all data 
when i try to do this i had an error

![Screenshot_20-11-2024_204933_api openf1 org](https://github.com/user-attachments/assets/5ad39128-dc01-4915-8d2a-f916ff483da3)
